db/schema.rb

```diff
@@ -1950,15 +1951,6 @@
 
   add_index "portal_teachers", ["user_id"], :name => "index_portal_teachers_on_user_id"
 
-  create_table "recent_collections_pages", :force => true do |t|
-    t.integer  "recent_project_id", :null => false
-    t.integer  "teacher_id",        :null => false
-    t.datetime "created_at",        :null => false
-    t.datetime "updated_at",        :null => false
-  end
-
-  add_index "recent_collections_pages", ["teacher_id"], :name => "index_recent_collections_pages_on_teacher_id"
-
```

Comment:
It is interesting that this is being removed here. But it looks like it was an early version of teacher_project_views and somehow it ended up in the schema.

Reply:
Here's the PT story: https://www.pivotaltracker.com/story/show/171546055